common/src/main/java/com/wynntils/features/overlays/AuraTimerOverlayFeature.java

```diff
@@ -90,42 +67,23 @@ protected AuraTimerOverlay() {
         }
 
         @Override
-        public void render(
-                PoseStack poseStack, MultiBufferSource.BufferSource bufferSource, float partialTicks, Window window) {
-            long remainingMs = getRemainingTimeUntilAura();
-            if (remainingMs <= 0) return;
-
-            String renderedString = "Aura: %.1fs".formatted(remainingMs / 1000f);
-
-            renderText(poseStack, bufferSource, renderedString);
+        public CustomColor getRenderColor() {
+            return textColor;
         }
 
         @Override
-        public void renderPreview(
-                PoseStack poseStack, MultiBufferSource.BufferSource bufferSource, float partialTicks, Window window) {
-            renderText(poseStack, bufferSource, "Aura: 3.2s");
+        public float getTextScale() {
+            return this.getHeight() / 15f;
         }
 
-        private void renderText(
-                PoseStack poseStack, MultiBufferSource.BufferSource bufferSource, String renderedString) {
-            BufferedFontRenderer.getInstance()
-                    .renderAlignedTextInBox(
-                            poseStack,
-                            bufferSource,
-                            renderedString,
-                            this.getRenderX(),
-                            this.getRenderX() + this.getWidth(),
-                            this.getRenderY(),
-                            this.getRenderY() + this.getHeight(),
-                            0,
-                            textColor,
-                            this.getRenderHorizontalAlignment(),
-                            this.getRenderVerticalAlignment(),
-                            textShadow,
-                            this.getHeight() / 15f);
+        @Override
+        public String getTemplate() {
+            return "{IF_STRING(MTE(AURA_TIMER; 0); CONCAT(CONCAT(\"Aura: : \"; STRING(AURA_TIMER:1)); \"s\"); \"\")}";
```

Comment:
Constant here as well. Also, it took me a while to realize what `MTE` is. I suggest we try to use `GT/GTE` which to me feels more standard. (In fact, if it were up to me we would remove MT/MTE, but if you like it I guess we can keep it)